# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/arcticfox/hotfix-xpcstubs.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- Arctic-Fox-46.1/xpcom/reflect/xptcall/md/unix/moz.build.vanilla	2025-08-12 23:35:52.147367188 +0200
+++ Arctic-Fox-46.1/xpcom/reflect/xptcall/md/unix/moz.build	2025-08-12 23:49:18.687684633 +0200
@@ -232,7 +232,7 @@
         'xptcstubs_ppc_openbsd.cpp',
     ]
 
-if CONFIG['OS_ARCH'] == 'Linux' and CONFIG['OS_TEST'] == 'sparc':
+if CONFIG['OS_ARCH'] == 'Linux' and CONFIG['OS_TEST'] in ('sparc','sparcv9'):
     SOURCES += [
         'xptcinvoke_asm_sparc_linux_GCC3.s',
         'xptcinvoke_sparc_solaris.cpp',
